desktop/apps/consignments2/client/actions.js

```diff
@@ -2,15 +2,31 @@ import { push } from 'react-router-redux'
 
 // Action types
 export const INCREMENT_STEP = 'INCREMENT_STEP'
+export const UPDATE_INPUTS = 'UPDATE_INPUTS'
 export const UPDATE_SUBMISSION = 'UPDATE_SUBMISSION'
 
 // Action creators
+export function createSubmission () {
+  return (dispatch) => {
+    // TODO: async request to convection to create submission
+    dispatch(updateSubmission()) // update state
+    dispatch(incrementStep()) // move to next step
+  }
+}
+
 export function incrementStep () {
   return {
     type: INCREMENT_STEP
   }
 }
 
+export function submitDescribeWork (values) {
+  return (dispatch) => {
+    dispatch(updateInputs(values)) // update the inputs in case we have to return
```

Comment:
Are you tracking this independently of `redux-form`'s form state? For every `reduxForm`-wrapped component a new reducer is created in redux, which should be the canonical source of truth.